browser.js

```diff
@@ -10,23 +10,25 @@ try {
 
 module.exports = nanotiming
 
+var inc = 0
```

Comment:
`inc` would need to be a global here to prevent multiple versions from interfering in the (global) performance timeline.